### PR TITLE
Create more complex outfit paradigm

### LINKED ISSUE #30: Use bjorn properly

### PR DESCRIPTION
**Pull request by horrible-little-slime**
Unfortunately obsoletes #76 but we do love new contributors
Resolves  #30

### CODE REVIEW

File: src/outfit.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    : {};
+}
+
+const chooseFamiliar = (options: MenuOptions = {}): Familiar =>
+  canInteract() && myInebriety() <= inebrietyLimit()
+    ? $familiars`Reagnimated Gnome, Temporal Riftlet`.find((f) => have(f)) ??
+      freeFightFamiliar(options)
+    : freeFightFamiliar(options);
+
+export function chooseQuestOutfit(quest: ChronerQuest, outfit: OutfitSpec = {}): OutfitSpec {
```
Comment (pstalcup) [inline src/outfit.ts:30]: can you make this `...outfitSpecs: OutfitSpec[]` instead, and merge it in this function? or is that too insane

File: src/outfit.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+const accessories = new Map<Item, () => number>([
+  [$item`mafia thumb ring`, () => (1 / 0.96 - 1) * get("valueOfAdventure")],
+  [
+    $item`time-twitching toolbelt`,
+    () =>
+      garboAverageValue(
+        ...$items`future drug: Muscularactum, future drug: Smartinex, future drug: Coolscaline`
+      ) / 18.2,
```
Comment (pstalcup) [inline src/outfit.ts:107]: what is 18.2 coming from

**Review by pstalcup — CHANGES_REQUESTED** [review]

File: src/outfit.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/outfit.ts:30]: yeah, I'll `reduce`

File: src/outfit.ts (hunk 2 above)
Comment (horrible-little-slime) [reply src/outfit.ts:107]: https://kol.coldfront.net/thekolwiki/index.php/Time-twitching_toolbelt
I'll add this as a comment

**Review by pstalcup — APPROVED** [review]

**Review by pstalcup — APPROVED** [review]

**Pull request merged**